src/view/calendar/calendar-view.jsx

```diff
@@ -71,26 +48,18 @@ export default function CalendarView() {
 				padding={{ all: 'medium' }}
 				style={{ overflowY: 'auto', position: 'relative' }}
 			>
-				<EventContext.Provider value={{ event, setEvent, action, setAction, toggleDeleteModal }}>
-					<CalendarStyle primaryCalendar={primaryCalendar} />
-					<Suspense fallback={<Text>Loading...</Text>}>
-						<CalendarComponent />
-					</Suspense>
-					{event && action === EventActionsEnum.EXPAND && (
-						<EventPanelView event={event} action={action} close={closePanel} />
-					)}
-					{event && action === EventActionsEnum.EDIT && (
-						<EventEditPanel event={event} action={action} close={closePanel} />
-					)}
-					{deleteModal && (
-						<DeleteEventModal
-							open={deleteModal}
-							event={event}
-							isInstance={isInstance}
-							onClose={toggleDeleteModal}
-						/>
-					)}
-				</EventContext.Provider>
+				<CalendarStyle primaryCalendar={primaryCalendar} />
+				<Suspense fallback={<Text>Loading...</Text>}>
```

Comment:
will it be better if we use shimmer instead of loading text ?

Reply:
It will, this need a different issue for that because it is not the only place which needs to be updated